test.js

```diff
@@ -78,6 +95,48 @@ it('should logging request of axios instance', () => axios.create()({
   )
 }))
 
+it('should log params of the request of axios instance', () => axios.create()({
+  method: 'GET',
+  url: 'http://example.com/',
+  params: {
+    name: 'value'
+  },
+  adapter: config => Promise.resolve({
+    status: 200,
+    statusText: 'BAR',
+    config
+  })
+}).then(() => {
+  debug.log.should.be.calledTwice()
+  debug.log.firstCall.should.be.calledWithExactly(
+    'GET http://example.com/?name=value'
+  )
+  debug.log.secondCall.should.be.calledWithExactly(
+    '200 BAR', '(GET http://example.com/?name=value)'
+  )
+}))
+
+it('should log params of the request of axios instance with special characters', () => axios.create()({
+  method: 'GET',
+  url: 'http://example.com/',
+  params: {
+    'parameter name': '&'
+  },
+  adapter: config => Promise.resolve({
+    status: 200,
+    statusText: 'BAR',
+    config
+  })
+}).then(() => {
+  debug.log.should.be.calledTwice()
+  debug.log.firstCall.should.be.calledWithExactly(
+    'GET http://example.com/?parameter%20name=%26'
+  )
+  debug.log.secondCall.should.be.calledWithExactly(
+    '200 BAR', '(GET http://example.com/?parameter%20name=%26)'
+  )
+}))
+
```

Comment:
I am not sure what happends if `url: 'http://example.com/?foo=bar'` and `params: { baz: 'qux' }`